.circleci/config.yml

```diff
@@ -42,10 +42,23 @@ jobs:
            sudo snap install *.snap --dangerous
            sudo nextcloud.manual-install admin admin
 
+     - run:
+         # Copy artifacts, install nodejs and publish download link in PR
+         command: |
+           sudo mkdir /artifacts/
+           sudo cp *.snap /artifacts/nextcloud.snap
+           wget -qO- https://deb.nodesource.com/setup_10.x | sudo -E bash -
+           sudo apt-get install -y nodejs
+           npm install circle-github-bot
+           sh $(pwd)/.circleci/publish-artifact.js
```

Comment:
You probably want to either execute this with node, or just execute it directly and let the shebang take effect.

Reply:
Thanks for the tip. I thought the shebang would be applied despite using `sh`.